packages/react-app/src/components/pages/Bounties/Bounty/index.tsx

```diff
@@ -18,6 +18,47 @@ import { BountyCollection } from '../../../../models/Bounty';
 import { baseUrl } from '../../../../constants/discordInfo';
 import { CustomerContext } from '@app/context/CustomerContext';
 import { useContext } from 'react';
+import axios from '@app/utils/AxiosUtils';
+import bountyStatus from '@app/constants/bountyStatus';
+import { useRouter } from 'next/router';
+
+const BountyActions = ({ bounty }: { bounty: BountyCollection }) => {
```

Comment:
Can you provide some documentation for `BountyActions`?

Reply:
Will make a note